alpm/src/types.rs

```diff
@@ -989,7 +989,7 @@ impl<'a> fmt::Debug for Group<'a> {
 }
 
 impl<'a> Group<'a> {
-    pub(crate) unsafe fn new<'b>(handle: &'b Alpm, ptr: *mut alpm_group_t) -> Group<'b> {
```

Comment:
I think this was done because I wasn't sure of the lifetime elision rules on &T instead of &self but looking it's the same thing. Looks fine in this cause but it's very easy to accidentally get the wrong lifetime when it elides and you're doing unsafe.